DependencyInjection/MonologExtension.php

```diff
@@ -41,9 +42,31 @@ class MonologExtension extends Extension
 
     private $swiftMailerHandlers = [];
 
-    private function levelToMonologConst($level)
+    private function levelToMonologConst($level, ContainerBuilder $container)
     {
-        return is_int($level) ? $level : constant('Monolog\Logger::'.strtoupper($level));
+        if ($level === null || is_int($level)) {
```

Comment:
Why do you allow returning  `null`?

Reply:
I figured not specifying a level will not pass it and cause the handler's default to be used. I will revert it